h_periodic/h_beat.py

```diff
@@ -55,14 +58,22 @@
         "fill-annotation-slim-first-batch": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_annotation_slim",
-            "schedule": crontab(hour="7", minute="*/3"),
-            "kwargs": {"batch_size": 250},
+            "schedule": crontab(hour="7", minute="*/2"),
+            "kwargs": {
+                "batch_size": 250,
+                "since": SLIM_TASK_SINCE,
+                "until": SLIM_TASK_UNTIL,
```

Comment:
Names match the task parameters: https://github.com/hypothesis/h/blob/main/h/tasks/annotations.py#L9